managed/ui/src/components/config/Storage/CreateBackup.test.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) YugaByte, Inc.
+//
+// Author: Nishant Sharma(nishant.sharma@hashedin.com)
+
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
```

Comment:
Putting this comment here to remind you to move to React Testing Library.

Reply:
For now, I'm removing the test file from this PR. I need to work on the `react testing library` thoroughly and once I get it properly I'll add the unit test cases. I'll keep this thing in mind.